cmd/nri-ecs.go

```diff
@@ -19,63 +19,51 @@ var (
 	integrationVersion = "1.3.1"
 )
 
-type argumentList struct {
+type ArgumentList struct {
```

Comment:
why exporting it?

Reply:
I forgot to change the package name of the `nri-ecs_test.go` which i just pushed :) . thanks for notice that!

Reply:
basically is being used by the tests since is needed in order to avoid parsing the same flags (global vars in the sdk) twice and panics for that reason.